src/Microsoft.AspNet.Diagnostics.Elm/project.json

```diff
@@ -17,5 +16,6 @@
                 "System.Threading": "4.0.11-beta-*"
             }
         }
-    }
+    },
+    "resource": ["Views/*.cshtml"]
```

Comment:
Or actually, what "bad" thing happens if it's left alone? Why mention it at all?